.github/workflows/lint.yml

```diff
@@ -13,6 +13,6 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
-      - run: yarn
+          node-version: '16.x'
```

Comment:
I think this GHA can be removed, since the gitlab pipeline has the same job

Reply:
Good point, removed